src/v2/guide/index.md

```diff
@@ -4,27 +4,27 @@ type: guide
 order: 2
 ---
 
-## What is Vue.js?
+## Vue.js, qu'est-ce que c'est ?
 
-Vue (pronounced /vjuː/, like **view**) is a **progressive framework** for building user interfaces. Unlike other monolithic frameworks, Vue is designed from the ground up to be incrementally adoptable. The core library is focused on the view layer only, and is very easy to pick up and integrate with other libraries or existing projects. On the other hand, Vue is also perfectly capable of powering sophisticated Single-Page Applications when used in combination with [modern tooling](single-file-components.html) and [supporting libraries](https://github.com/vuejs/awesome-vue#libraries--plugins).
+Vue (prononcé /vjuː/, comme le terme anglais **view**) est un **framework évolutif** pour construire des interfaces utilisateur. À la différence des autres frameworks monolithiques, Vue est conçu pour partir des bases et être progressivement pris en main. Le cœur de la bibliothèque est concentré uniquement sur la partie vue, et il est vraiment simple de l'intégrer avec d'autres bibliothèques ou projets existants. D'un autre côté, Vue est tout à fait capable de faire tourner des applications web monopages quand il est couplé avec [des outils modernes](single-file-components.html) et [des bibliothèques complémentaires](https://github.com/vuejs/awesome-vue#libraries--plugins).
```

Comment:
n°3 me voilà :)

Une traduction plus littérale de "from the ground up" pourrait être "du sol au plafond" mais dans le contexte présent, je traduirai plutôt par "dès l'origine".  @nyl-auster a bien compris l'idée, ça veut dire que ça a été pensé comme tel dès les premières fondations.